### DISCUSSION TITLE
Personal presentation ?

### DISCUSSION THREAD

**Discussion by brunetton** [jejellyroll-fr/fpdb-3#24] [General]
I'm Bruno Duyé, I currently live in the north of France. I'm developing software for 25 years, > 10 years of Python.

I'm passionate about poker, and I've done some open source poker contributions:
- https://brunetton.github.io/2024/07/20/pokertimer-open-source-poker-tournament-timer.html
- https://brunetton.github.io/2023/01/19/poker-hands-simple-rankings-charts.html

I'm also a musician: https://youtube.com/playlist?list=PL6QzQYkVJ_gCRKN_n9gUps8BbYDHkGJ49

I'm often in Paris for music, and work. @jejellyroll-fr if you live around, why don't meet and have some drink ?

**Comment by brunetton** [comment]
... probably wrong conversation, it should be an answer to https://github.com/jejellyroll-fr/fpdb-3/discussions/20 :s 

woops

**Comment by jejellyroll-fr** [comment]
Hello Bruno, 
thanks for the presentation, welcome to the community ;)
